EZSwiftExtensions.xcworkspace/xcshareddata/IDEWorkspaceChecks.plist

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Why do we need this file?

Reply:
I see. I still have to do the work to move it to swift 4.1 (its right now in swift 4.0 with XCode 9.2 as the build).

I presume this file is a no op for prior XCode builds?

Also while this is recommended in the release notes, I am still very skeptical of generated files. I presume an update to the unit tests would actually be captured here.